packages/react-components/react-card/src/components/Card/renderCard.tsx

```diff
@@ -8,5 +8,10 @@ import type { CardSlots, CardState } from './Card.types';
 export const renderCard_unstable = (state: CardState) => {
   const { slots, slotProps } = getSlots<CardSlots>(state);
 
-  return <slots.root {...slotProps.root} />;
+  return (
+    <slots.root {...slotProps.root}>
+      {slotProps.root.children}
+      {state.isSelectable && slots.select && <slots.select {...slotProps.select} />}
```

Comment:
lets be explicit and use ternary with null, shortcircuiting can lead to rendering issues
```suggestion
      {state.isSelectable && slots.select ? <slots.select {...slotProps.select} /> : null}
```

Reply:
Makes sense! Updated!